qrenderdoc/Windows/MainWindow.cpp

```diff
@@ -1408,8 +1408,9 @@ void MainWindow::on_action_Attach_to_Running_Instance_triggered()
 
 void MainWindow::on_action_Manage_Remote_Servers_triggered()
 {
-  // the manager deletes itself when all lookups terminate
-  RDDialog::show(new RemoteManager(m_Ctx, this));
+  RemoteManager *dialog = new RemoteManager(m_Ctx, this);
+  RDDialog::show(dialog);
+  delete dialog;
```

Comment:
It crashes when processing an enent after a target has been selected, deep down the stack:

1  ??
2  QTreeViewPrivate::hasVisibleChildren(QModelIndex const&) const
3  QTreeView::mouseDoubleClickEvent(QMouseEvent *)
4  QWidget::event(QEvent *)
5  QFrame::event(QEvent *)
6  QAbstractItemView::viewportEvent(QEvent *)
7  QTreeView::viewportEvent(QEvent *)
8  QCoreApplicationPrivate::sendThroughObjectEventFilters(QObject *, QEvent *)
9  QApplicationPrivate::notify_helper(QObject *, QEvent *)
10 QApplication::notify(QObject *, QEvent *)
11 QCoreApplication::notifyInternal2(QObject *, QEvent *)
12 QApplicationPrivate::sendMouseEvent(QWidget *, QMouseEvent *, QWidget *, QWidget *, QWidget * *, QPointer<QWidget>&, bool)
13 ??
14 ??
15 QApplicationPrivate::notify_helper(QObject *, QEvent *)
16 QApplication::notify(QObject *, QEvent *)
17 QCoreApplication::notifyInternal2(QObject *, QEvent *)
18 QGuiApplicationPrivate::processMouseEvent(QWindowSystemInterfacePrivate::MouseEvent *)
19 QGuiApplicationPrivate::processWindowSystemEvent(QWindowSystemInterfacePrivate::WindowSystemEvent *)
20 QWindowSystemInterface::sendWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>)
21 ??
22 g_main_context_dispatch
23 ??
24 g_main_context_iteration
25 QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>)
26 RDDialog::show                                                                                                             QRDUtils.cpp       358
27 MainWindow::on_action_Manage_Remote_Servers_triggered                                                                      MainWindow.cpp     1412
28 MainWindow::on_action_Attach_to_Running_Instance_triggered                                                                 MainWindow.cpp     1406
29 MainWindow::qt_static_metacall